packages/cli/internal/pkg/cli/account_activate.go

```diff
@@ -36,8 +37,13 @@ A new VPC will be created if not specified.`
 	publicSubnetsFlagDescription = `Do not create a NAT gateway or VPC endpoints, to lower operating costs. 
 You must enable the usePublicSubnets option in your project context if you use this flag.`
 	accountTagsDescription = `A list of comma separated tags to be applied to all AGC resources in this account
-(i.e. --tags "k1=v1","k2=v2"). Each key-value pair must be quoted as shown in the example,
+(e.g. --tags "k1=v1","k2=v2"). Each key-value pair must be quoted as shown in the example,
 otherwise the parsing will fail.`
+	subnetFlagDescription = `The list of private subnet IDs to use when specifying a VPC to run in. May only be used with the 
```

Comment:
Can we also add information about how this flag relates to `publicSubnets` flag?